product_operating_unit/tests/test_product_operating_unit.py

```diff
@@ -33,17 +30,15 @@ def setUp(self):
         # Create users
         self.user1_id = self._create_user('user_1',
                                           [self.group_stock_user],
-                                          self.company1,
                                           [self.ou1])
         self.user2_id = self._create_user('user_2',
                                           [self.group_stock_user],
-                                          self.company2,
                                           [self.b2b])
         self.product1.operating_unit_ids = [(6, 0, [self.ou1.id])]
         self.product2.operating_unit_ids = [(6, 0, [self.b2b.id])]
         self.product3.operating_unit_ids = [(6, 0, [self.ou1.id, self.b2b.id])]
 
-    def _create_user(self, login, groups, company, operating_units):
+    def _create_user(self, login, groups, operating_units):
```

Comment:
If no company is given it will work with the basic one, I guess that's the reason for this change. Now, I wonder, what about multi-company? How is this regarding operating units in the other tests of operating unit modules?

Reply:
I removed it so that we work with the basic company, yes. I think we are protected against multi-company because of the Python constraint.

Reply:
Ok, yes, I checked other tests in other operating units modules, and they seem not to address multi-company